integration/setup/setup.go

```diff
@@ -40,24 +45,85 @@ func installResources(ctx context.Context, config Config) error {
 	var err error
 
 	{
-		err := config.K8s.EnsureNamespaceCreated(ctx, namespace)
+		err := config.K8s.EnsureNamespaceCreated(ctx, key.Namespace())
 		if err != nil {
 			return microerror.Mask(err)
 		}
 	}
 
+	var latestOperatorRelease string
```

Comment:
Instead of using `Release.InstallOperator` we just install it here. This will get simpler once the operator is flat as we can use `project.Version()`.